app/containers/UserGeneratedContentRules.tsx

```diff
@@ -15,14 +15,16 @@ const styles = StyleSheet.create({
 		marginHorizontal: 30
 	},
 	bottomContainerText: {
-		...sharedStyles.textRegular,
-		fontSize: 13,
+		...sharedStyles.textMedium,
+		fontSize: 14,
+		lineHeight: 22,
 		textAlign: 'center'
 	},
-	bottomContainerTextBold: {
-		...sharedStyles.textSemibold,
```

Comment:
Not matching Figma.

You could do this
```
bottomContainerText: {
  ...sharedStyles.textMedium,
  fontSize: 14,
  lineHeight: 22,
  textAlign: 'center'
},
  bottomContainerTextUnderline: {
  textDecorationLine: 'underline'
}
```

```
style={[styles.bottomContainerText, styles.bottomContainerTextUnderline...
```

Reply:
Oops, I didn't realize medium text. Done! 

Thanks!